templates/search/list.html

```diff
@@ -0,0 +1,51 @@
+{% from 'blog/macros.html' import article_list with context %}
+{% extends 'layout.html' %}
+
+{% block title %}Search results{% endblock %}
+
+{% block extra_css %}
+<link rel=stylesheet href=//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css>
+<link rel=stylesheet href={{ url_for('static', filename='css/search.css') }}>
+<link rel=stylesheet href={{ url_for('static', filename='css/_article_list.css') }}>
+{% endblock %}
+
+{% block main %}
+<header>
+  <form action='{{ url_for('search') }}' method=get>
+    <input id=search type=search name=q placeholder="search for neomads and articles">
+    <input type=submit value=search>
+  </form>
+</header>
+
+<section>
+  {% if users %}
+  <article>
+    <h2>neomad finding</h2>
+    <ul>
+      {% for user in users %}
+      <li class=profiles>
+        <a href='@{{ user.slug }}'>
+          <h2 name=username>{{ user.username }}</h2>
+          <figure>
+            <img class=avatar-img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+          </figure>
+        </a>
+      </li>
+      {% endfor %}
+    </ul>
+  </article>
+  {% endif %}
+  
+  {% if articles %}
+  <article>
+    <h2>articles finding</h2>
+    <ul>
+      <li class=articles>
+        {{ article_list(articles=articles) }}
```

Comment:
`article_list` inside of one `li`? You will find the list of articles in one single `<li>`